Setup/RecurringData.php

```diff
@@ -4,11 +4,12 @@
 
 use Adyen\Payment\Helper\PaymentMethods;
 use Adyen\Payment\Helper\PaymentMethodsFactory;
-use Magento\Framework\Setup\InstallSchemaInterface;
+use Magento\Framework\Setup\InstallDataInterface;
 use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
 use Magento\Framework\Setup\SchemaSetupInterface;
 
-class Recurring implements InstallSchemaInterface
+class RecurringData implements InstallDataInterface
```

Comment:
Please update the related unit test file as it give the below error due to the name change

```
1) Adyen\Payment\Test\Unit\Setup\RecurringTest::testInstall
Error: Class "Adyen\Payment\Setup\Recurring" not found

/var/www/html/vendor/adyen/module-payment/Test/Unit/Setup/RecurringTest.php:29

```

Reply:
Should be fixed now.
![Screenshot 2024-02-19 at 16 23 50](https://github.com/Adyen/adyen-magento2/assets/2345875/c118b428-fbe7-40ce-b05c-1d60c34c9d6a)